src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -27,17 +27,16 @@
  */
 
 #include "squid.h"
+#include "debug/Stream.h"
 #include "helper/protocol_defines.h"
+#include "sbuf/SBuf.h"
+#include "sbuf/Stream.h"
 
-#include <cstdlib>
-#include <cstring>
+static const int DEBUG_SECTION=1;
+
+#include <chrono>
 #include <ctime>
-#include <sys/types.h>
-#include <sys/stat.h>
-#include <fcntl.h>
-#if HAVE_UNISTD_H
-#include <unistd.h>
-#endif
+#include <fstream>
```

Comment:
Remove as unused?

```suggestion
```

Reply:
Yes. Leftover from previous attempts to debug